sp800-63c/sec6_presentation.md

```diff
@@ -18,16 +18,25 @@ In the *back-channel* model, the subscriber is given an assertion reference to p
 
 In this model, the assertion itself is requested directly from the IdP to the RP, minimizing chances of interception and manipulation by a third party (including the subscriber themselves). 
 
-This method also allows the RP to query the CSP for additional attributes about the subscriber not included in the assertion itself, as back-channel communication can occur after the initial authentication transaction. 
+This method also allows the RP to query the CSP for additional attributes about the subscriber not included in the assertion itself, as back-channel communication can occur after the initial authentication transaction, or when the subscriber is no longer available. 
```

Comment:
This is definitely in scope as it's part of how the federation protocol can work. It is a machine to machine transaction but one about the current user, and I think it's important that we note it as part of the authentication process here. I also don't think we need to profile it beyond mentioning it like this.